pkg/services/govmomi/vcenter/clone.go

```diff
@@ -142,9 +146,19 @@ func Clone(ctx context.Context, vmCtx *capvcontext.VMContext, bootstrapData []by
 		if err != nil {
 			return errors.Wrapf(err, "error getting disk spec for %q", ctx)
 		}
+		log.V(4).Info("Got the following disks", "disks", diskSpecs)
```

Comment:
I'll move this log statement into the new if statement below this one.  I had added it originally only to log old and new disks.  Since I moved the data disk logic out of this if block, this is really needed below in the other if block.